### DISCUSSION TITLE
umami LXC DB access w/ pgAdmin

### DISCUSSION THREAD

**Discussion by vbalazs913** [tteck/Proxmox#3150] [Help Needed] [answered]
Hello fellow scripters!

I turn to you in hope of an answer regarding umami. I'm trying to migrate from docker to an LXC, and to do so I need to transfer my DB. I'm trying to connect to the PostgreSQL DB running inside the LXC container, but I can't seem to have any luck with it. All the postgres configurations, ports and firewall are in order, but nothing, pgAdmin can't connect. However, it perfectly works with the docker version.
Have any of you experienced this? If so, have you found any solution for this issue?

Thanks a lot, cheers!

**Comment by tteck** [comment]
Have you tried Adminer?

**Reply by vbalazs913** [reply]
Not yet, will do!

**Reply by vbalazs913** [reply]
Tried it, same with pgAdmin, Adminer says it can't connect. Ports are open, the service is running, but still no luck. Is there any special configuration being applied to the DB? The strange thing is that it works perfectly with a Docker instance of umami.

**Reply by tteck** [reply, marked as answer]
I use Adminer on my Umami LXC without issues. 🤷🏻‍♂️

**Reply by tteck** [reply]
![Screenshot 2024-06-10 12 36 42 PM](https://github.com/tteck/Proxmox/assets/21342670/83e3f3bb-cd65-4dcc-87c4-57a90f846d0b)

![Screenshot 2024-06-10 12 37 32 PM](https://github.com/tteck/Proxmox/assets/21342670/1d78627a-8f89-4696-9583-cd8cf376f719)
